sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -387,7 +363,7 @@ This project has adopted the [Microsoft Open Source Code of Conduct][coc]. For m
 <!-- LINKS -->
 [aad_authorization]: https://docs.microsoft.com/azure/cognitive-services/authentication#authenticate-with-azure-active-directory
 [aad_credential]: https://docs.microsoft.com/azure/cognitive-services/authentication#authenticate-with-azure-active-directory
-[api_reference_doc]: https://aka.ms/azsdk-java-textanalytics-ref-docs
```

Comment:
I would advice to leave the aka.ms link so we can keep updating it accordingly. Instead we can update the aka.ms link.
Looks like you have permission to change it

Reply:
updated